src/SessionSetup/ConnectToStadiaWidgetTest.cpp

```diff
@@ -0,0 +1,38 @@
+// Copyright (c) 2021 The Orbit Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#include <gtest/gtest.h>
+
+#include <QPushButton>
+#include <QRadioButton>
+#include <QTableView>
+
+#include "SessionSetup/ConnectToStadiaWidget.h"
+
+namespace orbit_session_setup {
+
+TEST(ConnectToStadiaWidget, IsSetActive) {
+  ConnectToStadiaWidget widget{};
+
+  auto* radio_button = widget.findChild<QRadioButton*>("radioButton");
```

Comment:
This is the radio button to choose for "Connect to Stadia instance" right? There are others below for the file loading and local profiling, right? Maybe add a very brief comment here?

Reply:
This is just the ConnectToStadiaWidget, so its always the same radio button. I added a more clarifying comment. ptal

Reply:
Ah thanks!